### PR TITLE
Write Changelog. Refs #614

### PR DESCRIPTION
**Pull request by koenedaele**

### CODE REVIEW

File: CHANGES.rst [hunk 1]
```diff
@@ -1,3 +1,48 @@
...
+
+- All requirements were updated to their latest versions. Python versions were
+  fixed to 2.7, 3.6, 3.7 and 3.8. If you made custom changes, you might have to
+  edit them. (#508, #519, #513, #566)
+- Instantiation of the SKOS registry was changed to work on a per request
+  basis. (#346, #490, #535)
+- Fixed a major issue with generating the expanded version of a concept. By
+  default the assumption was that concepts in a collection were also narrower
+  concepts of the collection's superordinate concept, but the implementation
+  for this was incomplete and contained buggy. This has been changed to an
```
Comment (cedrikv) [inline CHANGES.rst:26]: > was incomplete and contained buggy.

best replace with "was incomplete and buggy" or "was incomplete and contained bugs"

File: CHANGES.rst [hunk 2]
```diff
@@ -1,3 +1,48 @@
...
+  boolean attribute `infer_concept_relations`. When set to true, concepts in a
+  collection are considerd to be narrower concepts of that collections's
+  superordinate concept. This is especially important for a provider's
+  `expand` function and affects what is considered a narrower concept of a
+  concept that uses `thesaurus arrays` or `node labels`.
+- The docs were updated and now contain a part detailing what Atramhasis does
+  with some screenshots. (#495, #583, #440)
+- Default inclusion of skosprovider_heritagedata was removed because the
+  service is too unstable too often. (#537)
```
Comment (cedrikv) [inline CHANGES.rst:35]: > service is too unstable too often

service is unstable too often

File: CHANGES.rst [hunk 3]
```diff
@@ -1,3 +1,48 @@
...
+- Improve some SEO by adding canonicul URL's, open graph info, Twitter cards
+  and the ability to generate a sitemap through a script. (#530, #531, #496,
+  #497)
+- Clean up importing and exporting of conceptschemes to make it easier. (#452,
+  #475, #476, #495)
+- Provide a simple, printable version of a thesaurus tree. (#533, 532)
```
Comment (cedrikv) [inline CHANGES.rst:41]: forgot # for 532

**Review by cedrikv — CHANGES_REQUESTED** [review]
Maybe add a line about replacing bower with npm as package manager? Not sure if that's considered 'breaking'..

**Comment by koenedaele** [comment]
> Maybe add a line about replacing bower with npm as package manager? Not sure if that's considered 'breaking'..

Added one. Please review.

**Review by cedrikv — APPROVED** [review]

**Pull request merged**